src/components/Contacts/ContactFilters/ContactFilters.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import { MockedProvider } from '@apollo/client/testing';
+import React, { ReactElement } from 'react';
+import { ContactFilters } from './ContactFilters';
+import {
+  ContactFiltersQueryDefaultMocks,
+  ContactFiltersQueryEmptyMocks,
+  ContactFiltersQueryLoadingMocks,
+} from './ContactFilters.mock';
+
+export default {
+  title: 'ContactFilters',
+};
+
+const accountListId = '111';
+
+export const Default = (): ReactElement => {
+  return (
+    <MockedProvider mocks={ContactFiltersQueryDefaultMocks(accountListId)}>
+      <ContactFilters accountListId={accountListId} />
+    </MockedProvider>
+  );
+};
+
+export const Loading = (): ReactElement => {
+  return (
+    <MockedProvider mocks={ContactFiltersQueryLoadingMocks(accountListId)}>
+      <ContactFilters accountListId={accountListId} />
+    </MockedProvider>
+  );
+};
+
+export const Empty = (): ReactElement => {
+  return (
+    <MockedProvider mocks={ContactFiltersQueryEmptyMocks(accountListId)}>
+      <ContactFilters accountListId={accountListId} />
+    </MockedProvider>
+  );
+};
+
+Default.story = {
+  parameters: {
+    chromatic: { delay: 1000 },
```

Comment:
This still needed? You've got several of them.

Reply:
We probably don't need them... again, I found them in another Storybook file, so I thought it should be there for a reason.

Reply:
If you're using some sort of mocked provider, hopefully you don't need to delay it. There could still be issues though...